spec/controllers/teams_controller_spec.rb

```diff
@@ -308,17 +308,17 @@
 
   describe 'POST #perform_action_on_multiple_members' do
     it 'can delete ownership' do
-      team = Team.create! valid_attributes
-      another_user = FactoryBot.create :user
-      still_another_user = FactoryBot.create :user
+        team = Team.create! valid_attributes
+        another_user = FactoryBot.create :user
+        still_another_user = FactoryBot.create :user
 
-      team.owners << subject.current_user
-      team.owners << another_user
-      team.owners << still_another_user
+        team.owners << subject.current_user
+        team.owners << another_user
+        team.owners << still_another_user
 
-      expect {
-        post :perform_action_on_multiple_members, params: { id: team.id, members: [another_user.id, still_another_user.id], delete_ownership: "delete_ownership"}
-      }.to change(team.owners, :count).by(-2)
+        expect {
+          post :perform_action_on_multiple_members, params: { id: team.id, members: [another_user.id, still_another_user.id], delete_ownership: "delete_ownership" }
+        }.to change(team.owners, :count).by(-2)
       end
```

Comment:
this looks to be indented too much

Reply:
Yup, just fixed this. Weird autocorrect.